db/seeds.rb

```diff
@@ -61,9 +61,9 @@
 courses = []
 5.times do
   courses << Course.create!({
-    date_start: Date.today - rand(1000),
-    date_finish: Date.today - rand(1000),
-    time_start: Time.now,
+    date_start: DateTime.current - rand(1000),
+    date_finish: DateTime.current - rand(1000),
```

Comment:
make sure the date_finish value is after the date_start one. In fact you could add such a validation asap